### DISCUSSION TITLE
Weird plotting speed issue when using nvme as temp1 vs ramdisk

### DISCUSSION THREAD

**Discussion by shimpa1** [madMAx43v3r/chia-plotter#921] [Q&A]
So I have a number of servers with plenty of memory (usually 512GB).
I'm testing various scenarios and when I'm using a Samsung 970 nvme as temp1 I'm getting way better speeds in P3 vs when I'm using full ramdisk.
Basically P3 when using full ramdisk takes around 1000 secs and when using nvme for temp1 (and ramdisk as temp2) it takes maybe 400 seconds. That's a full 10 minutes difference and I know that it should actually be the other way around.
Did anyone have any similar experiences? This is weird...

**Comment by diaconom** [comment]
What speeds are your RAM ? And how have you mounted the RAM Drive? Ubuntu version?

**Reply by shimpa1** [reply]
It's a DDR4-2400, there's 512GB of it. 384GB ramdisk. It's mounted as a tmpfs

snip
tmpfs /mnt/ramdisk tmpfs defaults,size=384G 0 0
/snip

Ubuntu server 20.04

**Comment by diaconom** [comment]
My Ram is only DDR3-1060 and phase 3 is around 540secs !
